datalad/support/annexrepo.py

```diff
@@ -1382,6 +1382,12 @@ def get(self, files, remote=None, options=None, jobs=None, key=False):
         """
         options = options[:] if options else []
 
+        if self.config.get("annex.retry") is None:
+            options.extend(
+                ["-c",
+                 "annex.retry={}".format(
+                     self.config.obtain("datalad.annex.retry"))])
+
```

Comment:
Hmm - could it be you forgot to remove the respective part in `__init__` L304 or did I get the intention wrong?

Reply:
LOL, yes. thx!

Reply:
I pushed an update that fixes this.